src/components/ResultsCount.tsx

```diff
@@ -57,6 +57,8 @@ export function ResultsCount({ customCssClasses }: ResultsCountProps): JSX.Eleme
 function useResultsCount() {
   const isVertical = useAnswersState(state => state.meta.searchType) === SearchTypeEnum.Vertical;
   const results = useAnswersState(state => isVertical ? state.vertical : state.universal.verticals);
+  const offset = useAnswersState(state => isVertical ? state.vertical.offset: null) || 0;
```

Comment:
I don't think you need a ternary operator here!

Reply:
If you default it to 0 immediately after there's no point of having it set to null

Reply:
Ok!  I noticed that without the ternary operator, it still works on the test site but it fails universal search tests since `state.vertical` is `undefined`.  I assume in real use, `state.vertical` won't be undefined? In that case, should I just edit the tests to define `vertical`?

Reply:
Thanks!

Reply:
Fixed, and tests edited to conform.